test/e2e/scenarios/Cluster/cluster_terminal_condition.yaml

```diff
@@ -13,9 +13,9 @@ steps:
         name: cluster$RANDOM_SUFFIX
       spec:
         name: cluster$RANDOM_SUFFIX
-        nodeType: cache.t4g.medium
+        nodeType: cache.t4g.small
         aclName: open-access
-        numShards: 2
+        numShards: 1
```

Comment:
I dont think these changes are needed. `cache` is not valid node type for memorydb

Reply:
Ideally I agree with you, but this causes node limit exceed. It doesn't have this issue before, but this happens recently. It may cause by some code changes on service side. The modification makes test pass now, so we need to investigate the root cause.